src/utils/api.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+import Options from './options';
+
+const options = new Options();
+export default class API {
+  public async summaries() {
```

Comment:
I believe making this method to accept `start date and end date` would turn it more flexible for future implementations.